be/src/fs/fs_s3.cpp

```diff
@@ -182,7 +95,7 @@ std::shared_ptr<Aws::Auth::AWSCredentialsProvider> S3ClientFactory::_get_aws_cre
 
     if (!aws_cloud_credential.iam_role_arn.empty()) {
         // Do assume role
-        Aws::Client::ClientConfiguration clientConfiguration{};
+        Aws::Client::ClientConfiguration clientConfiguration = S3ClientFactory::getClientConfig();
```

Comment:
is this the only key change in this PR?

Reply:
Yes, other changes are in ut, also the same change.